flake.nix

```diff
@@ -412,6 +424,9 @@
           lsp-nls
           clippy
           rustfmt;
+        compileBenchmarks = (mkCraneArtifacts { }).benchmarks.overrideAttrs (old: {
+          buildPhase = "cargo bench --no-run";
```

Comment:
Ideally I would append from the overridden package, but I can’t just append to `buildPhaseCargoCommand` as it doesn’t exist in the final derivation.